help.txt

```diff
@@ -5,28 +5,28 @@
 
-== Non-Android Specific Issues
-For support issues not limited to the Android version of Anki, please visit the main http://ankisrs.net/docs/help.html[Anki help page].
+== Support
+If you were unable to find what you needed in the documentation above, please visit the relevant site below:
 
-== AnkiDroid Questions
-For questions and help concerning AnkiDroid, please visit the https://groups.google.com/forum/#!forum/anki-android[user forum], or send your questions to the forum's mailto:public-forum@ankidroid.org[public email address].
+Non-Android Specific Issues :: For support issues not limited to the Android version of Anki, please visit the main http://ankisrs.net/docs/help.html[Anki help page].
 
-== Bug Reports and Feature Requests
-For bug reports and feature requests, please search through the current list of open issues on the https://code.google.com/p/ankidroid/issues/list[AnkiDroid issue tracker], 
-and if there are no existing issues, create a new issue including as much information as possible. 
-Please also include the output of "debug info" from the AnkiDroid about dialog, which allows us to match your report with our internal crash report data.
+AnkiDroid Questions :: For questions and help concerning AnkiDroid, please visit the https://groups.google.com/forum/#!forum/anki-android[user forum], or you can send your questions to the forum's mailto:public-forum@ankidroid.org[public email address] if you don't want to register on the forum.
 
-To get the "debug info": 
+Bug Reports and Feature Requests :: For bug reports and feature requests, please search through the current list of open issues on the https://code.google.com/p/ankidroid/issues/list[AnkiDroid issue tracker], 
+and if there are no existing issues, create a new issue including as much information as possible. 
+For bug reports, please also include the output of "debug info" by following the steps below: 
 
 1. Open the navigation drawer by tapping the button on the top left of the screen
 2. Tap "settings"
 3. Tap "about AnkiDroid" at the bottom
 4. Tap the "copy debug info" button at the bottom (which copies "debug info" to the clipboard)
-5. Paste the contents of the clipboard to the issue
+5. If filling out the bug report on your mobile device, paste the contents of the clipboard directly to the issue. Alternatively, you can paste the contents of the clipboard into a new email, send it to yourself, then copy and paste that into the bug report from your computer.
 
 ++++++++++
 <img src="img/DebugInfo.png">
 ++++++++++
 
-== Contributing to AnkiDroid
+This "debug info" is important as it allows us to match your report with our internal crash report data.
```

Comment:
No the crash data is now on our private server. Of course if they set it to
never then we won't have the data, but it's not worth explaining in that
much detail
On 03/02/2015 8:47 pm, "snowtimeglass" <notifications@github.com> wrote:

> In help.txt
> <https://github.com/ankidroid/ankidroiddocs/pull/11#discussion_r23998451>:
>
> >
> >  ++++++++++
> >  <img src="img/DebugInfo.png">
> >  ++++++++++
> >
> > -== Contributing to AnkiDroid
> > +This "debug info" is important as it allows us to match your report with our internal crash report data.
>
> Does "our internal crash report data" mean the following data?
> http://ankidroid-triage.appspot.com/report_crashes
>
> Does this description fit even a situation in which an user sets "Error
> reporting modes" to "Never report"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ankidroid/ankidroiddocs/pull/11/files#r23998451>.
>